base/shared/src/main/scala/scalaz/tc/eq.scala

```diff
@@ -102,5 +102,8 @@ object Eq {
   def always[A]: Eq[A]                   = instanceOf[EqClass[A]]((_, _) => true)
   def never[A]: Eq[A]                    = instanceOf[EqClass[A]]((_, _) => false)
```

Comment:
off topic, but ... what is this for, again? isn't it unlawful to have a irreflexive `Eq`?